index.js

```diff
@@ -506,10 +504,6 @@ export class Placement {
       }
     }
 
-    // This logs all keywords found,
-    // even if they weren't found MIN_KEYWORD_OCCURRENCES
-    console.debug("EthicalAds detected keywords:", keywordHist);
```

Comment:
This is probably no longer necessary because now that keywords are exported they'll be available on the page with `ethicalads.detectedKeywords`.

Reply:
I used it for debugging, but I can always just look in the request 👍